packages/cfpb-forms/src/organisms/Multiselect.js

```diff
@@ -416,6 +417,20 @@ function Multiselect( element ) { // eslint-disable-line max-statements
       }
     } );
 
+    _headerDom.addEventListener( 'mouseup', function( event ) {
+      const target = event.target;
+
+      /* Check if we're over the down-arrow on the right side of the input.
+         Also check if the fieldset is open.
+         35 = width of the arrow on the right of the search input.
+         140 = the max-height value set in multiselect.less for the fieldset.
+      */
+      if ( event.layerX > target.offsetWidth - 35 &&
```

Comment:
Should we be using [offsetX](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/offsetX) instead of [layerX](https://developer.mozilla.org/en-US/docs/Web/API/UIEvent/layerX)? They operate the same but the former is in a spec while the latter is deprecated.

Reply:
Yes, good catch. It would be nice to find a better way to target the dropdown in the future…